docs/administration.platforms.md

```diff
@@ -91,13 +91,15 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Platform Danger Zone
 
-Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+Via the "Danger Zone" tab you can deprecate, undo a deprecation or delete your meshPlatform.
 
-Deactivation will make the meshPlatform not selectable anymore for new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. In order to immediately stop metering as well, the Metering Configuration needs to be removed. All settings of the platform will remain in meshStack by default. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform's tenants when a meshPlatform is deactivated.
+**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel.
```

Comment:
d: maybe also write that an indicator will tell people that those platforms are deprecated visually.